<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student_Wallet</class>
 <widget class="QDialog" name="Student_Wallet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #73C2FB; /* Ice Blue*/
    font-family: 'Segoe UI', sans-serif;
}</string>
  </property>
  <widget class="QLineEdit" name="Welcome">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
 {
    color: #FAFAD2;
    font-size: 24px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Welcome to Spartan Ride</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Spartan_Ride_App/SpartanLogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Images/person.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="student_name">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
 {
    color: #FAFAD2;
    font-size: 12px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Balamce">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
 {
    color: #FAFAD2;
    font-size: 12px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>125</y>
     <width>701</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="Student_pushbuttons">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="Wallet">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Wallet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RideHistory">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Ride History</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BookaRide">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Book Ride</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Profile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Feedback">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
   </property>
   <property name="text">
    <string>Feedback</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignOutButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>590</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
  <widget class="QLabel" name="Wallet_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
 {
    color: #191970;
    font-size: 24px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Wallet</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupbox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>491</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    background-color: #FFFACD;
    border: 2px solid #FFD700;
    border-radius: 10px;
    margin-top: 20px;
    font: bold 14px 'Segoe UI';
    color: #0B1D3A;
}


</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="CurrentBalance">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Current Balance :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PaymentMethod">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Payment Method :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PromoCode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Promo Code :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BillingEmail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Billing Email :</string>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentBalanceLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>65</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFACD;;  /* match your pale-yellow group-box */
color: black;               /* label text in black */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PaymentMethodLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFACD;;  /* match your pale-yellow group-box */
color: black;               /* label text in black */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="BillingEmailLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFACD;  /* match your pale-yellow group-box */
color: black;               /* label text in black */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PromoCodeLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFACD;  /* match your pale-yellow group-box */
color: black;               /* label text in black */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="HomeButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>590</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
